.gitignore

```diff
@@ -297,4 +297,5 @@ docs/metadata
 *input*txt*
 
 # Designer Files
-*.Designer.cs
\ No newline at end of file
+*.Designer.cs
+/test/CodeCoverage/codecoverage - Copy.bat
```

Comment:
do we need this line in the .gitignore?  It doesn't seem like a general file name that will be in most peoples' workspaces.